fused-effects-readline.cabal

```diff
@@ -41,6 +41,8 @@ library
     Control.Effect.Readline
   build-depends:
       base >= 4.12 && < 5
+    , directory ^>= 1.3.3.2
+    , filepath ^>= 1.4.2.1
```

Comment:
I wondered what was up with this—turns out my REPL script wasn’t hiding the packages that ship with the compiler. Mea culpa.